lib/order.rb

```diff
@@ -0,0 +1,69 @@
+require_relative 'menu'
+require_relative 'text'
+require 'rubygems'
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class Order
+  include Text
+  attr_reader :orders
+  def initialize
+    @orders = []
+  end
+
+  def greet
+    puts "Thank you for visiting our takeaway website."
+    puts "please take a look at our menu."
+  end
+
+  def present(menu)
+    menu.show
+  end
+
+  def choose_dish
+    puts "What would you like to order? Please enter a menu item number."
+    @menu_num = gets.chomp.to_i
+  end
+
+  def choose_how_many
+    puts "How many of dishes do you want?"
+    @quantities = gets.chomp.to_i
+  end
+
+  def cart(menu)
+    orders << menu.menu[@menu_num - 1].merge(quantities: @quantities)
+  end
+
+  def check_orders
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for check_orders is too high. [29.02/15]